### ISSUE TITLE
NullReferenceException on unknown PoV profession

### ISSUE THREAD

**Issue by Invisi** [baaron4/GW2-Elite-Insights-Parser#426]
**To Reproduce**
1. Get arcdps to produce an .evtc without the PoV profession
2. Tick "Add PoV profession to file name." in the settings
3. Try to parse the file
4. NullReferenceException

**Related code/variables**
log.LogData.PoV at https://github.com/baaron4/GW2-Elite-Insights-Parser/blob/ed08e738c63b81771111be435a16fb78014b0e1c/LuckParser/ProgramHelper.cs#L190
```
{LuckParser.Parser.ParsedData.AgentItem}
[...]
    Name: "UNKNOWN"
    Prof: null
    Toughness: 0
    Type: NPC
```

See attached file, names have been edited. Running a build of the current master branch.

[20190818-164328.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/3513333/20190818-164328.zip)

**Comment by EliphasNUIT** [comment]
I'll fix it in next.

How do you generate arc dps logs without PoVs though? If you know the trick you may want to contact Delta regarding it, the absence of PoV is not normal.

**Comment by Invisi** [comment]
No idea, it just happened at random in some of my WvW evtcs. I'll let you (and him) know if I ever figure out how to reproduce.

**Comment by EliphasNUIT** [comment]
fixed in 719ebf7

**Comment by EliphasNUIT** [comment]
Fixed in 2.16.2.0
